### PR TITLE
Speakers info

### PR DESCRIPTION
**Pull request by keiakihito**
## Updated 
Adding speakers' information.

## Background
Speakers provided their information.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,17 +60,21 @@ <h3>イベント日時・場所</h3>
                     <article>
                         <h3>スピーカー</h3>
                         <div class="speaker">
+                            <!-- TODO : プロフィール画像を受け取ったら更新-->
                             <img src="images/speaker1.png" alt="" class="image"/>
```
Comment (kkatsumiPCC) [inline index.html:64]: Got it! Let's wait until the picture is delivered.

**Review by kkatsumiPCC — COMMENTED** [review]
Let's wait until the pictures come.

File: index.html (hunk 1 above)
Comment (keiakihito) [reply index.html:64]: I got profile pics and updated it. Can you check and merge it if it looks fine?

Comment (kkatsumiPCC) [reply index.html:64]: Yes it looks good to me!

**Pull request merged**